connectd/multiplex.c

```diff
@@ -622,6 +622,7 @@ static bool handle_custommsg(struct daemon *daemon,
 			     const u8 *msg)
 {
 	enum peer_wire type = fromwire_peektype(msg);
+	// fixme use allow_even_custom_messages config here
```

Comment:
Makes sense to allow everything through here, *but* be careful: we should change the semantics of `custommsg` hook to allow it to reply "result": "accept" (and terminate traversal of hooks if it does).  If nobody does this for an even message, we should tell connectd to send an error and close.